autorest/authorization.go

```diff
@@ -258,3 +261,21 @@ func (egta EventGridKeyAuthorizer) WithAuthorization() PrepareDecorator {
 	}
 	return NewAPIKeyAuthorizerWithHeaders(headers).WithAuthorization()
 }
+
+// BasicAuthorizer implements authorization for Basic authentication.
+type BasicAuthorizer struct {
+	token string
+}
+
+// NewBasicAuthorizer creates a new BasicAuthorizer with the specified token.
```

Comment:
dont we need a username also on top of the token/password ?

Reply:
From the [docs](https://docs.microsoft.com/en-us/rest/api/azure/devops/?view=azure-devops-rest-5.0) it looks like all that's required is the token.